fischkapp/src/components/AddNewCard/AddNewBackCard.tsx

```diff
@@ -32,6 +33,7 @@ export const AddNewBackCard = ({
     };
 
     onSetFlashCards([...flashCards, newFlashCard]);
```

Comment:
**nitpick:** It should look like this:

```js
const createdCard = postFlashCards(newFlashCard);
onSetFlashCards([...flashCards, createdCard]);
```

Then you won't need to reload page to have new card visible.